meantemp/terra_meantemp.py

```diff
@@ -125,22 +118,26 @@ def process_message(self, connector, host, secret_key, resource, parameters):
                 create_datapoint_with_dependencies(connector, host, secret_key, "IR Surface Temperature",
                                                    (centroid_lonlat[1], centroid_lonlat[0]), time_fmt, time_fmt,
                                                    dpmetadata, timestamp)
+            else:
+                self.log_info(resource, "not a number")
```

Comment:
With the current fullfield image including the black boundary rows, we're going to see a log of these, right?

Reply:
possibly, although that's only if numpy.nanmean() returns a NaN - that function should take the mean of any partial boundary rows so we'll only see this message if the boundary is entirely black.

For partial (half field) scans we'll potentially see hundreds. I think I'll modify this to count the number of else cases found and print one summary message at the end instead ("43 plots returned NaN")